libtcmu_config.c

```diff
@@ -149,11 +151,19 @@ static void tcmu_conf_set_options(struct tcmu_config *cfg, bool reloading)
 		 * 4, default /var/log/
 		 */
 		if (!tcmu_get_logdir())
-			tcmu_logdir_create(cfg->log_dir_path);
+			tcmu_logdir_create(cfg->log_dir_path, false);
 		else
 			tcmu_warn("The logdir option from the tcmu.conf will be ignored\n");
 	} else {
-		tcmu_warn("The logdir option is not supported by dynamic reloading for now!\n");
+		/*
+		 * Here we asume that users want to change the
+		 * log_dir_path without considering tha priority
+		 * mentioned above.
+		 */
+		tcmu_logdir_create(cfg->log_dir_path, true);
+		ret = tcmu_create_file_output(tcmu_get_log_level(), TCMU_LOG_FILENAME, true);
+		if (ret < 0)
+			tcmu_err("create file output error \n");
```

Comment:
Let's change this to:

Could not change log path to %s.\n, new log dir path

I would say also report the error code, but create_file_output is a mess of -1 and -errno.